cf-runagent/cf-runagent.c

```diff
@@ -400,7 +401,7 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
                 bool ret = LogEnableModulesFromString(optarg);
                 if (!ret)
                 {
-                    exit(EXIT_FAILURE);
+                    DoCleanupAndExit(EXIT_SUCCESS);		/* return of false indicates that help was requested, so not failure */
```

Comment:
make sense now? I added an @return comment to the LogEnableModulesFromString() definition.

Reply:
Well, I don't know, I think this comment is still confusing. First of all, I would write `ret == false` instead of `return of false` to make it clear that it's referring to the bool above. And "help was requested" is still confusing. Who needs help? Who is going to provide help? How was help requested? What is help?